internal/pkg/manager/spod/bindata/spod.go

```diff
@@ -352,6 +357,72 @@ semodule -i /opt/spo-profiles/selinuxd.cil
 							},
 						},
 					},
+					{
+						Name:            "metrics-proxy-spod",
+						Image:           metricsImage,
+						ImagePullPolicy: v1.PullAlways,
+						Args: []string{
+							"--secure-listen-address=0.0.0.0:8443",
+							"--upstream=http://127.0.0.1:8081/",
+							"--v=10",
+						},
+						Resources: v1.ResourceRequirements{
+							Requests: v1.ResourceList{
+								v1.ResourceMemory:           resource.MustParse("32Mi"),
+								v1.ResourceCPU:              resource.MustParse("50m"),
+								v1.ResourceEphemeralStorage: resource.MustParse("10Mi"),
+							},
+							Limits: v1.ResourceList{
+								v1.ResourceMemory:           resource.MustParse("128Mi"),
+								v1.ResourceCPU:              resource.MustParse("150m"),
+								v1.ResourceEphemeralStorage: resource.MustParse("20Mi"),
+							},
+						},
+						SecurityContext: &v1.SecurityContext{
+							AllowPrivilegeEscalation: &falsely,
+							RunAsUser:                &userRootless,
+							SELinuxOptions: &v1.SELinuxOptions{
+								// TODO(pjbgf): Use a more restricted selinux type
+								Type: "spc_t",
+							},
```

Comment:
This probably doesn't need `spc_t`. Let's leave that empty so it uses the default `container_t`

Reply:
Removed as suggested :heavy_check_mark: